docs/disabled_features.md

```diff
@@ -18,6 +18,20 @@ They may be enabled by default in future versions.
 The `@` modifier lets you specify the evaluation time for instant vector selectors,
 range vector selectors, and subqueries. More details can be found [here](querying/basics.md#modifier).
 
+## Negative offset in PromQL
+
+This negative offset is disabled by default since it breaks the invariant
+that PromQL does not look ahead of the evaluation time for samples.
+
+`--enable-feature=promql-negative-offset`
+
+In contrast to the positive offset modifier, the negative offset modifier lets
+one shift a vector into the future.  An example in which one may want to use a
```

Comment:
```
For example, the following expression returns the value of
`http_requests_total` 5 minutes in the past relative to the current     
query evaluation time: 
```

We could something more similar to this wording, do you have something to propose?

Reply:
That sounds confusing to me because it just sounds like you are taking a value from 5 minutes in the past (could also refer to a positive offset), and it's unclear that time-shifting happens. I would for a start just replace "future" with "past" in the line I commented on here, because that seems like a clear mistake.

Reply:
foo offset -5m takes foo 5 minutes in the future.

Reply:
I guess the vector is moved to the past but the vector "selector" is moved to the future.

Reply:
Correct :)